backend/src/main/scala/cromwell/backend/wdl/WriteFunctions.scala

```diff
@@ -27,8 +30,15 @@ trait WriteFunctions extends IoFunctionSet with AsyncIoFunctions {
     }
   }
 
+  private val relativeToLocal = {
+    val initial = System.getProperty("user.dir")
+    if (initial.endsWith("/")) initial else initial + "/"
+  }
+
+  def relativeToAbsolutePath(pathFrom: String): String = if (new File(pathFrom).isAbsolute) pathFrom else relativeToLocal + pathFrom
```

Comment:
Could just add this as an "ioFunction" ? I need exactly this somewhere else for `ExpressionTool`s